src-backend/api/serializer.py

```diff
@@ -44,11 +44,30 @@ def to_representation(self, instance):
         return ret
 
 
+class PageListSerializer(serializers.ListSerializer):
+    class Meta(object):
+        model = models.Page
+
+    def update(self, instance, validated_data):
+        page_mapping = {page.id: page for page in models.Page.objects.all()}
```

Comment:
aren't these both page mappings? if so, the variables should be renamed to better reflect where they came from e.g. `cur_mapping` and `new_mapping`